snuba/datasets/storages/transactions.py

```diff
@@ -91,6 +92,7 @@
         EventIdColumnProcessor(),
```

Comment:
Isn't this the one we should be replacing?

Reply:
I didn't realize that logic had been moved to a processor. However it's not that simple, because we still want the EventId processor to run, so the return values to Sentry are correct. The UUID processor I added is only used in the condition clauses.

Reply:
I see. We are never actually formatting `trace_id` like `event_id` though (perhaps we should be). Does this still need to be handled in the UUID column processor then? The treatment of these columns at the moment is quite inconsistent.

Reply:
I think ideally the UUID column processor would handle the formatting in all parts of the query, rather than rely on the user adding two separate processors (the one that handles the conditions part and the one that handles the other parts of the query) in the correct order and on the same columns which seems easy to get wrong.

Reply:
I can make that change, but it will make this processor even more complicated. Do you think that's worth doing as part of this PR or should I make a separate task for it?

Reply:
Up to you, whichever is easier. I'd suggest to leave a TODO at least though.